translations/de/hints/bryukh.html

```diff
@@ -17,23 +17,23 @@
         Ich benötige etwas Hilfe, um mit der Mission weiterfahren zu können
     </div>
     <div class="answer">
-        Versuche <a href="https://wiki.python.org/moin/HowTo/Sorting/#Key_Functions">den Schlüssel-Parameter "key" für das Sortieren</a>.
+        Benutze <a href="https://wiki.python.org/moin/HowTo/Sorting/#Key_Functions">den Schlüsselparameter (key) für das Sortieren</a>.
```

Comment:
**`(key)`** to **`"key"`**

May bad...
the quotes around `key` have been correct